README.md

```diff
@@ -44,7 +80,37 @@ Some interesting code locations are:
 - [`src/propositions/NullPropositions.hs`](src/propositions/NullPropositions.hs): Operators for the ternary logic with `null`.
 We provide truthtables for the logic in [meta/Truthtable.md](meta/Truthtable.md).
 The implementation is based on our implementation for propositional logic in [`src/propositions/Propositions.hs`](src/propositions/Propositions.hs).
-You can inspect the truth tables for the ternary logic by uncommenting the respective line (`showTruthtables`) in the `main` function in [`app/Main.hs`](app/Main.hs) and running the project again.
+You can inspect the truth tables for the ternary logic by uncommenting the respective line (`showTruthtables`) in the `main` function in [`app/Main.hs`](app/Main.hs) and running the demo again.
+
+
+## Please cite as
+```tex
+@inproceedings{BST+:ESECFSE21,
+	author = {Paul Maximilian Bittner and Alexander Schulthei\ss{} and Thomas Th{\"{u}}m and Timo Kehrer and Jeffrey M. Young and Lukas Linsbauer},
+	title = {{Feature Trace Recording}},
+	booktitle = {Proc.\ Europ.\ Software Engineering Conf./Foundations of Software Engineering (ESEC/FSE)},
+	location = {Athens, Greece},
+	publisher = {ACM},
+	address = {New York, NY, USA},
+	year = 2021,
+	month = AUG,
+	note = {To appear}
+}
+```
```

Comment:
Shouldn't the citation refer to the artifact instead of the paper? Maybe add the URL or Zenodo DOI?

Reply:
We don't want people to cite the artefact but not the paper. Citing artefact and paper separately is weird, too. In any case, the paper is the primary artifact that should be cited. Also I never saw dedicated citations for only the artefacts of a paper but not the paper. I will add the DOI to the article.